VotingApplication/VotingApplication.Web/Scripts/Directives/VoteChoice/VoteChoice.html

```diff
@@ -0,0 +1,17 @@
+﻿<div ng-switch="poll.PollType">
+
+    <div ng-switch-when="Basic">
+        <basic-vote-choice choices="choices"></basic-vote-choice>
+    </div>
+
+    <div ng-switch-when="Multi">
+        <multi-vote-choice choices="choices"></multi-vote-choice>
+    </div>
+
+    <div ng-switch-when="UpDown">
+        <up-down-vote-choice choices="choices"></up-down-vote-choice>
+    </div>
+
+    <div ng-switch-when="Points">POINTS</div>
```

Comment:
Yup. Tried something with sliders, but the angular material sliders don't currently do what we want (partial disablement). As noted in the PR comment, Points are are still in progress, as I'm not sure what they're going to look like yet...